### ISSUE TITLE
Cannot call start a second time

### ISSUE THREAD

**Issue by stalleyj** [RuntimeTools/appmetrics#336]
If you try calling start(), stop() and then start() again, it fails and the process ends.

**Comment by baens** [comment]
Not just can't call start/stop/start. If you do this start/stop/start cycle, things break. Like `setInterval`.

Here is a simple log output of running start/stop/start in a JS file with the debug output set in appmetrics.

[debug.txt](https://github.com/RuntimeTools/appmetrics/files/607599/debug.txt)

**Comment by baens** [comment]
Also, here is a fine demonstrating that start/stop/start cycle breaking `setInterval`

```
var appmetrics = require('appmetrics');

function start() {
  console.log('--------------------START----------------------------');
  appmetrics.start();
}

function stop() {
  console.log('---------------------STOP----------------------------');
  appmetrics.stop();
}

setInterval(() => console.log('tick'), 1500);

const run = [stop, start];

function timeout() {
  console.log('timeout');
  if (run.length == 0) return;
  const next = run.shift();
  next();
  setTimeout(timeout, 5000);
}

setTimeout(timeout, 5000);
```

**Comment by stalleyj** [comment]
This will be investigated further and fixed in a future release

**Comment by baens** [comment]
So, I just did a quick dive of disabling plugins, and I believe it has to to with the [Node Loop Plugin](https://github.com/RuntimeTools/appmetrics/blob/master/src/plugins/node/loop/nodeloopplugin.cpp).

What it could be no idea, but at least when I remove that from my repository it seems to work.

**Comment by sjanuary** [comment]
Now fixed by #348
